### DISCUSSION TITLE
My Vercel account cannot be connected to Github

### DISCUSSION THREAD

**Discussion by FaizanMK** [vercel/community#6553] [Account Issue]
### Question

 When I try to log in to Vercel and click continue with GitHub, I receive this error: **"There is already an account associated with your GitHub email address. Please login with your email and password then link your GitHub account."** Then when I try to log in with my email and enter my email I receive this error: **"We are unable to complete your signup."** @amyegan @tomocchino @swarnava ?
I have already sent an email to registration@vercel.com but didn't get any response from there.

**Comment by Bahriddin-Boboyev** [comment]
Currently, this error is observed with some people. **Vercel** is aware of this bug and please expect it to be fixed soon.

**Comment by swarnava** [comment]
Please email us at registration@vercel.com and our team will triage this shortly

**Reply by FaizanMK** [reply]
As I have mentioned I have already sent an email to the given email address, but haven't gotten any response from them.
